test/lifecycle.test.js

```diff
@@ -142,6 +143,9 @@ beforeEach(async function () {
   sails.testServer = testServer;
   sails.testPlayer = testPlayer;
   sails.testServerConfig = testServerConfig;
+  sails.testServer.config = testServerConfig;
```

Comment:
>   sails.testServer.config = testServerConfig;
So this makes me extremely nervous. This will make tests pass when accessing server.config even when populate isn't called.

Could we just get away with await SdtdServer.find(sails.testServer.id).populate('config')? or maybe create a new server at the top of the tele test?